mock/py/mockbuild/buildroot.py

```diff
@@ -538,7 +538,14 @@ def _setup_devices(self):
             self.root_log.debug("kernel version == %s", kver)
             for i in devFiles:
                 src_path = "/" + i[2]
-                chroot_path = self.make_chroot_path(i[2])
+
+                if util.USE_NSPAWN:
+                    chroot_path = os.path.join('/dev/mock/',
+                                               self.config['root'],
+                                               i[2])
```

Comment:
WARN WARN! This is **the hack**, we shouldn't maintain another set of directories under `/dev/mock`.  But since `/dev/mock/...` "startswith" `/dev`, systemd-nspawn doesn't object...